lib/ecmascript.ts

```diff
@@ -1573,19 +1824,26 @@ export function GetISO8601Calendar() {
   return new TemporalCalendar('iso8601');
 }
 
-export function CalendarFields(calendar, fieldNames) {
+// TODO: should (can?) we make this generic so the field names are checked
```

Comment:
This seems interesting :) We should try this in another PR.

Reply:
👍